python/kwiver/vital/util/pybind11.h

```diff
@@ -37,84 +37,99 @@
 #include <pybind11/pybind11.h>
 
 namespace kwiver {
+
 namespace vital {
+
 namespace python {
 
 /*
  * NOTE: These classes and macros have just been lifted from pybind11 and
  * rewritten to use the Python interpreter's standard PyGILState_* API, because
  * of the inherent compatibility issues in the pybind11 version. In the future,
- * we would like to get similar classes and macros merged into upstream pybind11
+ * we would like to get similar classes and macros merged into upstream
+ *pybind11
  * so that they become standard.
```

Comment:
```suggestion
 * pybind11 so that they become standard.
```

(Note: comments that get wrapped will often need to be rewrapped by hand. IIRC, there's an option for this but it is intentionally off because automatic reflow can make things worse. See also previous comment on enforcing style vs. applying style 🙂.)